src/panel/settings/settings_wm.vala

```diff
@@ -95,16 +96,17 @@ public class WindowsPage : Budgie.SettingsPage {
         combo_layouts.set_id_column(0);
 
         /* Hook up settings */
+        gnome_wm_settings = new GLib.Settings("org.gnome.mutter");
         budgie_wm_settings = new GLib.Settings("com.solus-project.budgie-wm");
-        budgie_wm_settings.bind("attach-modal-dialogs", switch_dialogs,  "active", SettingsBindFlags.DEFAULT);
```

Comment:
Should maintain use of budgie_wm_settings instead of using gnome_wm_settings for attach-modal-dialogs and edge-tiling. We can just additionally change the gnome_wm_settings value in our switch_dialogs and switch_settings functions.